src/Nelmio/Alice/Fixtures/Builder/Methods/ReferenceRangeName.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+use Nelmio\Alice\Instances\Collection;
+
+final class ReferenceRangeName implements MethodInterface
+{
+    /**
+     * @var array $matches
+     */
+    private $matches = [];
+
+    /**
+     * @var \Nelmio\Alice\Instances\Collection
+     */
+    private $objects;
+
+    /**
+     * ReferenceRangeName constructor.
+     * @param \Nelmio\Alice\Instances\Collection $objects
+     */
+    public function __construct(Collection $objects)
+    {
+        $this->objects = $objects;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function canBuild($name)
+    {
+        return 1 === preg_match('#\{@([0-9a-zA-Z\._\-]+)(\*?+)\}#i', $name, $this->matches);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function build($class, $name, array $spec)
+    {
+        $fixtures = [];
+
+        $referenceName = $this->matches[1];
```

Comment:
maybe for the sake of readability, add a inline comment about what could be `$referenceName`, ex `// 'car' from engine_{@car1}`